NEWS.md

```diff
@@ -2,6 +2,7 @@
 
 * Status date set on newly created instance records (MODINVSTOR-509)
 * Item status date is set for newly created item records (MODINVSTOR-508)
+* Allow generation of HRID values with and without leading zeroes (MODINVSTOR-661)
```

Comment:
Thank you for adding a news entry that is meaningful from the user's perspective

Reply:
Thank you for review.